go-selinux/selinux_linux.go

```diff
@@ -237,20 +235,21 @@ func readConfig(target string) string {
 	scanner := bufio.NewScanner(in)
 
 	for scanner.Scan() {
-		line := strings.TrimSpace(scanner.Text())
-		if len(line) == 0 {
+		str := bytes.TrimSpace(scanner.Bytes())
```

Comment:
Was there a reason to rename `line` to `str`? (it's slightly confusing because `str` is not a _string_ but a `[]byte` now)

Reply:
Yes, that was not called for, and introduces some additional changes that are not needed.

Reverted back.